.operations/operations-manual.md

```diff
@@ -1,10 +1,10 @@
 # Operations Manual - Organizing and Maximizing Our Work
-Building a community and knowledge by efficiently handling issues
+Building knowledge and a community by efficiently handling issues
 
-## Handling issues and PR
+## Handling issues and PRs
 
 <br/>
-In a nutshell, every issue and PR should get tagged by one of our core team and routed to the person who specializes in the related topic. This person then, will warmly welcome and kick the discussion shortly (hopefully within 48 hours). The goal of each issue/PR is to learn new thing that might improve our repo and try joining the opener to our forces.
+In a nutshell, every issue and PR should get tagged by one of our core team and routed to the person who specializes in the related topic. This person then, will warmly welcome the contributor and then kick off the discussion (hopefully within 48 hours). The goal of each issue/PR is to learn new thing that might improve our repo and try to include the contributor in our army.

```

Comment:
Isn't army a militant word?

Reply:
Yes but so is force in my opinion. Maybe a better word could be used instead.
I'm thinking "family" but it lacks the original feeling of "being unstoppable" 😂
Now thinking of it again I actually prefer "force" if no better alternative could be found.